.github/workflows/scripts/linux_arm64/rh8/build_mhlo.sh

```diff
@@ -36,6 +36,10 @@ export PATH=/catalyst/llvm-build/bin:/opt/_internal/cpython-${PYTHON_VERSION}.${
 # Patch MHLO for correct linking
 sed -i -e 's/LINK_LIBS PUBLIC/LINK_LIBS PUBLIC MLIRDeallocationUtils/g' mlir/mlir-hlo/deallocation/transforms/CMakeLists.txt
 
+# Patch MHLO cmake dependency
+patch -p1 /catalyst/.github/workflows/patches/mhlo-Add-PassesIncGen-in-transforms-CMakeList.patch /catalyst/mlir/mlir-hlo/mhlo/transforms/CMakeLists.txt
```

Comment:
If possible, I would add this to the Makefile and not the CI build scripts. This would allow us to also run it on our computers. Finally, if this is not yet upstream, you should contribute it to the tensorflow repo :)